### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by mhk8**
fixed changes requested in old repo

### CODE REVIEW

**Comment by mhk8** [comment]
image_2020_12_08T14_55_56_297Z
image_2020_12_08T14_55_38_315Z
image_2020_12_08T14_55_21_949Z
image_2020_12_08T14_55_08_005Z

**Review by speshori — APPROVED** [review]

File: src/views/Profileview.tsx [hunk 1]
```diff
@@ -33,46 +29,32 @@ function Profileview() {
...
             {name}
           </div>
           <div className="profile-username">
             Username:&nbsp;
-            {user}
+            { user}
           </div>
           <div className="profile-data">
             User Since:&nbsp;
-            {rtime ? <TimeDisplay time={rtime} /> : null}
```
Comment (dvkm) [inline src/views/Profileview.tsx]: Why are you undoing my changes :/

**Review by dvkm — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 2]
```diff
@@ -37,6 +38,102 @@ function App() {
   return (
     <Router>
       <AppHeader loggedIn={loggedIn} setUserId={setUserId} />
+      <Switch>
```
Comment (dvkm) [inline src/App.tsx]: This should not be moved here.

File: src/views/Styles.css [hunk 3]
```diff
@@ -1,196 +0,0 @@
-.gradient-border-home {
```
Comment (dvkm) [inline src/views/Styles.css:1]: Why delete this

File: src/views/Leaderboard.tsx [hunk 4]
```diff
@@ -61,9 +59,9 @@ function Leaderboard() {
               {loading ? <CircularProgress color="secondary" /> : null}
               {transactions.map((transaction) => (
                 <TableRow className={classes.tableRow} component={Link} to={`/profile/${transaction.id}`} hover>
-                  <TableCell style={{ color: `${transaction.color}` }}>{transaction.id}</TableCell>
-                  <TableCell style={{ color: `${transaction.color}` }}>{transaction.name}</TableCell>
-                  <TableCell style={{ color: `${transaction.color}` }}>{transaction.balance}</TableCell>
```
Comment (ak2253) [inline src/views/Leaderboard.tsx]: Can you give me an explanation why you are redoing my changes?

File: src/views/Profileview.tsx [hunk 5]
```diff
@@ -19,8 +17,6 @@ function Profileview() {
   const [user, setUser] = useState('');
   const [rtime, setRtime] = useState('');
   const [tickets, setTickets] = useState('');
-  const [profileURL, setProfileURL] = useState('');
-  const [color, setColor] = useState('');
```
Comment (ak2253) [inline src/views/Profileview.tsx]: This is required for customization so can you explain why you decided to delete it throughout Profileview.tsx?

**Review by ak2253 — CHANGES_REQUESTED** [review]

**Review by dvkm — CHANGES_REQUESTED** [review]

**Review by ak2253 — APPROVED** [review]

File: src/App.tsx (hunk 2 above)
Comment (dvkm) [reply src/App.tsx]: Please fix this.

**Review by dvkm — CHANGES_REQUESTED** [review]

**Pull request closed without merging**